packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -211,6 +262,20 @@ export class ApexReplayDebug extends LoggingDebugSession {
     this.printToDebugConsole(
       nls.localize('session_started_text', this.logContext.getLogFileName())
     );
+    // If the projectPath isn't set then don't bother with heap dump processing
+    if (this.projectPath) {
```

Comment:
I was torn between sending the boolean of whether or not checkpoints were enabled versus using an undefined projectPath. The fact is, if the path is somehow undefined then we're not going to be able to get the heap dumps anyways. Rather than passing around the enabled status, on the extension side if checkpoints aren't enabled then it passes over an undefined project path.